lib/create-file-decompression-interface-context.yml

```diff
@@ -0,0 +1,14 @@
+rule:
+  meta:
+    name: create File Decompression Interface context
```

Comment:
Thoughts on adding `on Windows` to make it more clear?

Reply:
Thanks! Renamed the rule in https://github.com/mandiant/capa-rules/pull/808/commits/ac09516cca968e54d6b5764405337cba3f290c21